core/src/main/java/hudson/model/Job.java

```diff
@@ -369,6 +369,7 @@ public EnvVars getCharacteristicEnvVars() {
         env.put("JENKINS_SERVER_COOKIE",SERVER_COOKIE.get());
         env.put("HUDSON_SERVER_COOKIE",SERVER_COOKIE.get()); // Legacy compatibility
         env.put("JOB_NAME",getFullName());
+        env.put("SHORT_JOB_NAME",getName());
```

Comment:
IMHO the variable name is not perfect. It would be great to have something self-documented